src/stores/campaigns.store.ts

```diff
@@ -80,12 +85,20 @@ export const useCampaignStore = create<CampaignStore>()((set, getState) => ({
       })
     );
   },
-  removeCampaign: (campaignId: string) => {
+  removeCampaign: async (campaignId: string) => {
```

Comment:
Ah my bad naming has caused some confusion here. Whoops- the `removeCampaign` function here is simply to remove an _already deleted_ campaign from the store - it is called whenever the user's list of campaigns changes. We need a new function `deleteCampaign` to handle the firestore logic.

Reply:
Done 👍